### ISSUE TITLE
latest tf_sparse only support tf2.*?

### ISSUE THREAD

**Issue by caowencai** [CrawlScript/tf_sparse#1]
https://github.com/CrawlScript/tf_sparse/blob/d3066fb49fcd383238564a201907e6cd4572e942/tf_sparse/sparse_matrix.py#L12

As showed above,  tensorflow.python.ops.linalg.sparse only exists in tf==2.* version，and there exists import error at tf_sparse here when using tf_geometric with tf=1.5.0

**Comment by hujunxianligong** [comment]
Yes. Currently tf_sparse only supports tf 2.x. We will consider to make it
compatible with tf 1.x in the future.

**Comment by caowencai** [comment]
> 

so also, tf_geometric only supports tf 2.x?  So upset.

**Comment by caowencai** [comment]
> Yes. Currently tf_sparse only supports tf 2.x. We will consider to make it compatible with tf 1.x in the future.
> […](#)
> Message ID: ***@***.***>

so also, tf_geometric only supports tf 2.x? So upset.

**Comment by hujunxianligong** [comment]
I think if you're using tf 1.x with eager execution, you should upgrade to tf 2.x.
It's more efficient in many cases.

**Comment by caowencai** [comment]
> 

You're right. But out seving system is using tf1.x, 意大利炮也毁灭不了 哈哈

**Comment by caowencai** [comment]
> I think if you're using tf 1.x with eager execution, you should upgrade to tf 2.x. It's more efficient in many cases.

You're right. But our seving system is using tf1.x, which cannot be destroyed by Artillery of Italy ，哈哈
